### PR TITLE
feat(pie-docs): DSW-1259 added components L-P

### PR DESCRIPTION
**Pull request by LTurns**
## Describe your changes (can list changeset entries if preferable)

[Added] - Overview pages for Components L - P

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] If it is a `PIE Docs` change, I have reviewed the Docs site preview
- [ ] If it is a component change, I have reviewed the Storybook preview
- [ ] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: apps/pie-docs/src/components/link/link.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+---
+
+## Overview
+The purpose of links is to provide a visual and interactive way for users to explore and navigate through an interface. By clicking or tapping on a link, users can quickly access additional information or move to a different location, enhancing their browsing experience and enabling efficient interaction.
+
+Links are often used to connect various pages, sections, or external resources, creating navigation menus and directing users to related content. They also enable interactions like opening new tabs or windows.
+
+{% contentPageImage {
+    src:"../../assets/img/components/link/overview.svg",
+    alt: "Two icon buttons that are horizontally stacked together."
```
Comment (FayeCarter) [inline apps/pie-docs/src/components/link/link.md:14]: this might be wrong

Comment (LTurns) [reply apps/pie-docs/src/components/link/link.md:14]: 😂 thank you!

**Review by jamieomaguire — DISMISSED** [review]

**Review by FayeCarter — DISMISSED** [review]

**Review by raoufswe — DISMISSED** [review]

File: apps/pie-docs/src/components/list-item/list-item/list-item.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+eleventyNavigation:
+    key: 'List Item'
+    parent: list-item
+---
+
+## Overview
+The purpose of list items is to present individual pieces of content or data within a larger list, enabling users to easily scan, navigate, and interact with the information. It helps organise and structure content in a user-friendly and intuitive manner.
+
+List items can be styled with various visual elements such as icons, illustrations, thumbnail (among others). They are commonly used in content listings or any situation where multiple items need to be displayed in a structured way.
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/list-item/list-item/list-item.md:10]:
```suggestion
List items can be styled with various visual elements such as icons, illustrations and thumbnails (among others). They are commonly used in content listings or any situation where multiple items need to be displayed in a structured way.
```

File: apps/pie-docs/src/components/numeric-stepper/numeric-stepper-form/numeric-stepper-form.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+---
+eleventyNavigation:
+    key: 'Numeric Stepper - Form'
+    parent: numeric-stepper
+---
+
+## Overview
+The purpose of numeric steppers as form fields is to offer users flexibility in inputting numeric values, maintaining a consistent form field appearance in form interfaces. This allows users to easily adjust and enter numerical data with ease.
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/numeric-stepper/numeric-stepper-form/numeric-stepper-form.md:8]:
```suggestion
The purpose of numeric steppers as form fields is to offer users flexibility in inputting numeric values, while maintaining a consistent form field appearance in form interfaces. This allows users to easily enter and adjust numerical data.
```

File: apps/pie-docs/src/components/numeric-stepper/numeric-stepper-form/numeric-stepper-form.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+Numeric steppers [form] resemble the structure of classic form fields, but allow for precise value entry through the integrated text input while also offering the convenience of incremental adjustments using the provided buttons.
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/numeric-stepper/numeric-stepper-form/numeric-stepper-form.md:10]: This sentence is a bit wordy, in my opinion, we should be aiming for simpler words and phrases that are easy for anyone to understand on a Friday afternoon 😄

File: apps/pie-docs/src/components/progress-bar/progress-bar.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+---
+eleventyNavigation:
+    key: 'Progress Bar'
+    parent: Components
+---
+
+## Overview
+The purpose of progress bars is to inform users about the progress of a task, reducing uncertainty, and providing a visual sense of completion. It helps manage expectations, improves user experience, and gives users a sense of control and understanding.
+
+You can use progress bars can in your products to show the advancement uploads, software installations, form submissions, loading processes or any task that involves sequential progress.
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/progress-bar/progress-bar.md:10]:
```suggestion
You can use progress bars in your products to show the progress of uploads/downloads, software installations, form submissions, loading processes or any task that involves sequential progress.
```

File: apps/pie-docs/src/components/progress-stepper/progress-stepper.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
+---
+eleventyNavigation:
+    key: 'Progress Stepper'
+    parent: Components
+---
+
+## Overview
+The purpose of progress steppers is to guide users through a series of steps, visually indicating their progress and completion status. It helps users navigate complex processes, maintain a sense of orientation, and ensures a smoother user journey.
+
+Progress steppers are commonly used in multi-step forms, onboarding processes, checkout flows or any situation where a complex task is divided into manageable steps. They enhance the user experience by providing a visual roadmap and aiding in task completion.
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/progress-stepper/progress-stepper.md:10]: I think using the words "continuous" and "discrete" might be a good way of distinguishing the progress-bar from the progress-stepper. What do you think?

File: apps/pie-docs/src/components/numeric-stepper/numeric-stepper-form/numeric-stepper-form.md (hunk 4 above)
Comment (LTurns) [reply apps/pie-docs/src/components/numeric-stepper/numeric-stepper-form/numeric-stepper-form.md:10]: I agree, what about something like:

`Numeric steppers [form] allow a user to enter precise values, either by inputting text or by using the increment or decrement buttons.`

File: apps/pie-docs/src/components/numeric-stepper/numeric-stepper-form/numeric-stepper-form.md (hunk 3 above)
Comment (LTurns) [reply apps/pie-docs/src/components/numeric-stepper/numeric-stepper-form/numeric-stepper-form.md:8]: this small change got me looking at this 😄 https://www.grammarly.com/blog/while-vs-whilst/#:~:text=Whilst%20and%20while%20are%20two,That's%20the%20main%20difference.

File: apps/pie-docs/src/components/numeric-stepper/numeric-stepper-form/numeric-stepper-form.md (hunk 4 above)
Comment (xander-marjoram) [reply apps/pie-docs/src/components/numeric-stepper/numeric-stepper-form/numeric-stepper-form.md:10]: Yeah that's great!

**Review by xander-marjoram — APPROVED** [review]

**Review by FayeCarter — APPROVED** [review]
